lib/tic_tac_toe.rb

```diff
@@ -0,0 +1,11 @@
+
+module TicTacToe
+  class App
+    def say_hello
+      puts "Hello, welcome to Tic Tac Toe"
+    end
+  end
+
+  app = App.new
+  app.say_hello
```

Comment:
This means that whenever this module is loaded, it will start the game and print out `Hello, welcome to Tic Tac Toe`. This may not be desirable when you are running tests or CI. Perhaps you can change it in your next PR.